src/EnergyPlus/StandardRatings.cc

```diff
@@ -7085,15 +7086,15 @@ namespace StandardRatings {
         // Formats
 
         if (CompTypeNum == CoilDX_CoolingSingleSpeed) {
-            if (state.dataHVACGlobal->StandardRatingsMyCoolOneTimeFlag2) {
+            if (state.dataHVACGlobal->StandardRatingsMyCoolOneTimeFlag3) {
```

Comment:
When `...Flag2` was already in use here when it was added for the 2023 ratings. So move this one to `...Flag3`.